sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -498,6 +518,39 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
       else:
         return_type = typehints.Any
       state.stack[-pop_count:] = [return_type]
+    elif opname == 'CALL':
+      pop_count = 1 + arg
+      # Keyword Args case
+      if state.kw_names is not None:
+        if isinstance(state.stack[-pop_count], Const):
+          from apache_beam.pvalue import Row
+          if state.stack[-pop_count].value == Row:
```

Comment:
does it make sense to move this code into a helper and reuse in CALL_FUNCTION_KW processing?

Reply:
This is doable but a little finnicky since the `CALL` version and the `CALL_FUNCTION_KW` version have slightly different accessed indices in the stack